.github/workflows/linux-eic-shell.yml

```diff
@@ -852,32 +859,32 @@ jobs:
           export DETECTOR_CONFIG=${DETECTOR}_${{ matrix.detector_config }}
           export LD_LIBRARY_PATH=$PWD/install/lib:$LD_LIBRARY_PATH
           export JANA_PLUGIN_PATH=$PWD/install/lib/EICrecon/plugins:/usr/local/plugins
-          $PWD/install/bin/eicrecon $JANA_OPTIONS -Ppodio:output_file=rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root sim_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4hep.root -Pacts:WriteObj=true -Pacts:WritePly=true -Pplugins=janadot,janatop $(<${{ github.workspace }}/.github/janadot.groups)
+          $PWD/install/bin/eicrecon $JANA_OPTIONS -Ppodio:output_file=rec_dis_${{matrix.beamelectron}}x${{matrix.beamproton}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root sim_dis_${{matrix.beamelectron}}x${{matrix.beamproton}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4hep.root -PLOWQ2:TaggerTrackerTrajectories:electron_beamE=${{matrix.beamelectron}} -Pacts:WriteObj=true -Pacts:WritePly=true -Pplugins=janadot,janatop $(<${{ github.workspace }}/.github/janadot.groups)
```

Comment:
What is the blocker for propagating beam energy from afterburner through DD4hep to EDM4hep metadata?

Reply:
The afterburner PR is still open and has been for a while, my implementation broke a couple of ways it could be configured previously but I wanted to know if anyone was actually using them or if they could be swapped to a different approach. 

If the hepmc file has the information in its RunAttributes dd4hep was updated so that it passes this information through the simulation. I tested this but it might have needed some additional flag, I'll have to go back and check.

The main hold up will be a metadata service in EICrecon, requested in #1336 which I believe is on the todo list of @nathanwbrei

Reply:
I don't think we can hand this off like this to production. What about the constants defined in DD4hep https://github.com/eic/epic/blob/7c7f26f34371e70754ad582487b32f79acb0ac4c/compact/fields/beamline_10x100.xml#L6 ?

Reply:
What would be the problem with having this in production?

We would then need to separately set the xml for each eicrecon configuration in the workflows which I believe isn't currently done as the reconstruction behaves the same, although users are probably more likely to keep the xml identical between simulation and reconstruction in their own scripts. Both solutions are stop gaps for having the metadata present in the file but having a command line config parameter with the ability to override the meta data makes more sense to me longer term.

In a final solution what would the precedence of the configuration be?

1. Command line config
2. Metadata
3. Detector geometry

Reply:
The issue is that this implies that production and users have to populate every such config option.

I understand your point about xml, my assumption was that this might be already incorporated in the production. Looks like I'm likely wrong.

What do you think about re-using logic from https://github.com/eic/EICrecon/blob/fa7377c9261724b849e9e6f54a581ebe1fe42522/src/algorithms/reco/InclusiveKinematicseSigma.cc#L42-L53 ?

Reply:
This is also used like that in the far forward region: https://github.com/eic/EICrecon/blob/fa7377c9261724b849e9e6f54a581ebe1fe42522/src/algorithms/fardetectors/MatrixTransferStatic.cc#L57-L60

Adding a configuration option and expecting someone downstream to provide the right value is not workable in the short term (i.e. 24.06).

Reply:
While it has some downsides I will concede the far-forward solution is currently best from the user point of view.